src/dodal/devices/watsonmarlow323_pump.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class WatsonMarlow323PumpEnable(str, Enum):
+    DISABLED = "Disabled"
+    ENABLED = "Enabled"
+
+
+class WatsonMarlow323PumpDirection(str, Enum):
+    CLOCKWISE = "CW"
+    COUNTER_CLOCKWISE = "CCW"
+
+
+class WatsonMarlow323PumpState(str, Enum):
+    STOPPED = "STOP"
+    STARTED = "START"
+
+
+class WatsonMarlow323Pump(StandardReadable):
+    """Watson Marlow 323 Pump device"""
+
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.enabled = epics_signal_rw(
+            WatsonMarlow323PumpEnable,
+            read_pv=prefix + "DISABLE",
+            write_pv=prefix + "DISABLE",
+        )
+
+        self.direction = epics_signal_rw(
+            WatsonMarlow323PumpDirection,
+            read_pv=prefix + "INFO:DIR",
+            write_pv=prefix + "SET:DIR",
+        )
+        self.state = epics_signal_rw(
+            WatsonMarlow323PumpState,
+            read_pv=prefix + "INFO:RUN",
+            write_pv=prefix + "SET:RUN",
+        )
+        self.speed = epics_signal_rw(
+            float, read_pv=prefix + "INFO:SPD", write_pv=prefix + "SET:SPD"
+        )
+        self.limit_high = epics_signal_rw(
+            float, read_pv=prefix + "SET:SPD.DRVH", write_pv=prefix + "SET:SPD.DRVH"
+        )
+        self.limit_low = epics_signal_rw(
+            float, read_pv=prefix + "SET:SPD.DRVL", write_pv=prefix + "SET:SPD.DRVL"
+        )
```

Comment:
Limits (and units, precision...) will propagate with the speed signal once [this change is merged](https://github.com/bluesky/ophyd-async/pull/137), these signals shouldn't be required: as is, it's not clear what they are the limits of, but when the change is merged they'll become metadata on the speed signal.

Reply:
Ok I'll remove these when bluesky/ophyd-async#137 is merged

Reply:
Rebased and fixed in 84824ca3d5a6f0e30f8cc00fa40fec3cea06a2aa.